giftsonhelloworld.py

```diff
@@ -0,0 +1,3 @@
+

```

Comment:
So whenever you right a python file follow the below things.
The python file should start like this alwats
"""
whatever your script does definition goes here.
"""
if its a script use a function and call a function always.

Reply:
do all the print in single line.

Reply:
@belikegt  please create a folder of your name and in it add this file